### DISCUSSION TITLE
How to set palette for Custom.qss?

### DISCUSSION THREAD

**Discussion by Marocco2** [PCSX2/pcsx2#8069] [General]
Hi!
I've made a Custom.qss borrowed from another project but I don't know how I should set up icon theme and it looks like this:
![image](https://user-images.githubusercontent.com/3944248/217202150-fd0506f1-cdca-4e45-b0f6-93319d33eac0.png)

Also, if I switch from a dark theme and going back, icons appearance change to the previous theme:

https://user-images.githubusercontent.com/3944248/217202920-a1b13d4c-b372-447f-a53a-a4fd01c01234.mp4

**Comment by RedDevilus** [comment]
Problem seem to be when 6.4 Qt was used and doesn't seem to refresh properly.

**Reply by Marocco2** [reply]
Well, looking to the code of Custom in MainWindow.cpp, setPalette isn't present, so I assume it will inherent what did happen before you set a custom theme.

**Comment by TSnatch72** [comment]
Totally unrelated, but i really like your theme. Would you mind sharing it?

**Reply by Marocco2** [reply]
I adapted it from here: https://github.com/witalihirsch/qBitTorrent-fluent-theme
[Custom.qss.txt](https://github.com/PCSX2/pcsx2/files/13597643/Custom.qss.txt)

Delete the `.txt` extension and it should work
